### PR TITLE
Rendering a List of Meals

### PR DESCRIPTION
**Pull request by lkk95**
I created a Meal component and render it by mapping over a static meals array.
Also I added a test for the component and some (very) basic styling for homepage.

(belongs to ticket #7)

### CODE REVIEW

**Review by mbosselmann — COMMENTED** [review]
Great! 👍 I left some remarks for you to consider before moving to the next step. :)

File: src/components/Layout/Layout.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import Header from '../Header';
+
+import Main from './styled.js';
+
+export default function Layout({children}) {
+	return (
+		<>
+			<Header />
+			<Main>{children}</Main>
+			<Footer />
```
[the remark this answers is not in the document]
Comment (lkk95) [reply src/components/Layout/Layout.js]: Thanks for your review, Mareike! :blush:
I deleted all unnecessary files and components. Also I deleted the comment for the static array.

File: src/components/MealCard/MealCard.spec.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		render(<h2>{titletext}</h2>);
+		const title = screen.getByText(titletext);
+
+		const categorytext = 'Dinner';
+		render(<p>{categorytext}</p>);
+		const category = screen.getByText(categorytext);
+
+		expect(title).toBeInTheDocument();
+		expect(category).toBeInTheDocument();
+	});
```
[the remark this answers is not in the document]
Comment (lkk95) [reply src/components/MealCard/MealCard.spec.js]: So if I understand it correct then you can render the component with the props that you want to test and then get them by the 'getByText' method?
This is definitely shorter than my version, so thank you for your remark!:thumbsup:

File: src/pages/index.js [hunk 3]
```diff
@@ -1,15 +1,56 @@
...
+	];
+
 	return (
 		<Layout>
 			<Helmet>
 				<title key="title">My Project</title>
 				<meta key="description" name="description" content="This is my project" />
 			</Helmet>
-			<h1>Home</h1>
+			<h1>Your planned meals</h1>
```
[the remark this answers is not in the document]
Comment (lkk95) [reply src/pages/index.js:47]: I adjusted it to my working title 😊

**Review by mbosselmann — APPROVED** [review]
Checked your changes and I approve! 👍

**Pull request merged**